README.md

```diff
@@ -1,8 +1,63 @@
+<<<<<<< HEAD
 # JanusGraph Docker images
 
+Repository for building and publishing [JanusGraph][JG] docker images.
+
 ## License
 
 JanusGraph Docker images are provided under the [Apache 2.0
 license](APACHE-2.0.txt) and documentation is provided under the [CC-BY-4.0
 license](CC-BY-4.0.txt). For details about this dual-license structure, please
 see [`LICENSE.txt`](LICENSE.txt).
+
+## Building ##
+
+The `build-images.sh` script will build the docker images for all the Dockerfiles in the versioned
+folder directories.
+
+## Publishing ##
+
+The `push-images.sh` script will push the docker images for all the versioned folders in the repo.
+
+Prior to publishing, you'll need to login to [Docker Hub][DH] using the `docker login` command.
+
+## Configuration ##
```

Comment:
I think it would be a good idea to also explain that the image by default uses Berkeley DB and Lucene within the same container and maybe where data is stored by default.